clang/test/CIR/CodeGen/temporaries.cpp

```diff
@@ -14,9 +14,9 @@ void f() {
   !E();
 }
 
-//      CIR: cir.func private @_ZN1EC1Ev(!cir.ptr<!rec_E>) extra(#fn_attr)
+//      CIR: cir.func private @_ZN1EC1Ev(!cir.ptr<!rec_E>) cxx_ctor<!rec_E, default> extra(#fn_attr)
 // CIR-NEXT: cir.func private @_ZN1EntEv(!cir.ptr<!rec_E>) -> !rec_E
-// CIR-NEXT: cir.func private @_ZN1ED1Ev(!cir.ptr<!rec_E>) extra(#fn_attr)
+// CIR-NEXT: cir.func private @_ZN1ED1Ev(!cir.ptr<!rec_E>) cxx_dtor<!rec_E> extra(#fn_attr)
```

Comment:
Sounds like we could also drop "cxx": `dtor<!rec_E>` is probably good enough? No need to change this right now, let's converge on the syntax first (see other comment).